### ISSUE TITLE
How to revert back to bash, ctrl+alt+t not opening bash 

### ISSUE THREAD

**Issue by tanmayhinge** [vercel/hyper#5437]
Following the below instructions, i was able to set hyper as my default termainl that worked like a charm, however  i wanted to switch back to bash as my default terminal... so when i use the shortcut to open terminal bash should open but that doesnt seem to work now...

@baborlelefan Try running to command below to add Hyper to your config. If that doesn't work, can we please see the output from the command you ran?

sudo update-alternatives --install /usr/bin/x-terminal-emulator x-terminal-emulator /opt/Hyper/hyper 50

_Originally posted by @chaseleinart in https://github.com/vercel/hyper/issues/2764#issuecomment-374437229_

**Comment by LabhanshAgrawal** [comment]
did you try https://github.com/vercel/hyper/issues/2764#issuecomment-435697407

**Comment by tanmayhinge** [comment]
> did you try [#2764 (comment)](https://github.com/vercel/hyper/issues/2764#issuecomment-435697407)

Yes, for some reason the x-terminal file is missing in my system

**Comment by LabhanshAgrawal** [comment]
what do you mean by bash not opening
it'd open inside hyper if that's your shell
do you mean gnome-terminal etc.
